src/content/docs/en/guides/upgrade-to/v6.mdx

```diff
@@ -97,6 +97,30 @@ The following deprecated features are no longer supported and are no longer docu
 
 Some deprecated features may temporarily continue to function until they are completely removed. Others may silently have no effect, or throw an error prompting you to update your code.
 
+### `Astro` in `getStaticPaths()`
+
+Until now, it was possible to access the `Astro` object in `getStaticPaths()`. However, despite being typed the same as the `Astro` object accessible in the frontmatter, this object only had `site` and `generator` properties.
+
+In Astro 6.0, accessing `Astro.site` or `Astro.generator` within `getStaticPaths()` will log a warning. Accessing any other property will throw an error. In a future major version, accessing `Astro.site` or `Astro.generator` will also throw an error.
```

Comment:
```suggestion
Astro 6.0 deprecates this object to avoid confusion and improves error handling when attempting to access `Astro.` values that are unavailable. Using `Astro.site` or `Astro.generator` within `getStaticPaths()` will now log a deprecation warning, and accessing any other property will throw a specific error with a helpful message. In a future major version, this object will be removed entirely, and accessing `Astro.site` or `Astro.generator` will also throw an error.
```

Reply:
Updated with a suggestion!